java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JEnum.java

```diff
@@ -48,59 +47,48 @@ public String getType() {
         return this.type;
     }
 
-    private String sanitizeEnumEntry(String str) {
-        str = str.replace(" ", "_");
-        str = str.replace("/", "_");
-        return str;
+    private static String sanitizeEnumEntry(final String str) {
+        String res = str;
+        res = res.replace(" ", "_");
+        res = res.replace("/", "_");
+        return res;
```

Comment:
Should be able to do this in one pass

```suggestion
        return str.replaceAll("[\s/]", "_");
```